query/README.md

```diff
@@ -1,64 +1,64 @@
+You can build a KeenQueryClient by just providing a KeenProject. Note that for query purposes, the write key is not required. It is therefore OK and normal to provide ```null``` argument for the write key, unless that same KeenProject will be used for publishing events as well.
 ```java
 KeenProject queryProject = new KeenProject("<project id>", "<write key>", "<read key>");
-KeenQueryClient queryClient = new TestKeenQueryClientBuilder(queryProject).build();
+KeenQueryClient queryClient = new KeenQueryClient.Builder(queryProject).build();
 ```
 Optionally, users can also specify a HTTP Handler, base URL, or JSON Handler:
 ```java
-KeenQueryClient queryClient = new TestKeenQueryClientBuilder(queryProject)
+KeenQueryClient queryClient = new KeenQueryClient.Builder(queryProject)
 		.withHttpHandler(httpHandler)
 		.withJsonHandler(jsonHandler)
 		.withBaseUrl(baseURL)
 		.build();
 ```
 ### Using the KeenQueryClient to send Queries
-The most simple way that users can use the KeenQueryClient to send queries is as follows. These methods take only the required query parameters as input, and the user receives a very specific Integer or Double response type. Please note that we strongly encourage users to pass in the Timeframe parameter, but it can be null.
+The most simple way that users can use the KeenQueryClient to send queries is as follows. These methods take only the required query parameters as input, and the user receives a very specific ```long``` or ```double``` response type. Please note that we strongly encourage users to pass in the Timeframe parameter, but it can be null.
 ```java
-Long count = queryClient.count("<event_collection>", new RelativeTimeframe("this_year"));
-Long countUnique = queryClient.countUnique("<event_collection>", "<target_property>", new AbsoluteTimeframe("2012-08-13T19:00:00.000Z","2015-06-07T19:00:00.000Z"));
-Double minimum = queryClient.minimum("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double maximum = queryClient.maximum("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double average = queryClient.average("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double median = queryClient.median("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double percentile = queryClient.percentile("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double sum = queryClient.sum("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
+long count = queryClient.count("<event_collection>", new RelativeTimeframe("this_week"));
+long countUnique = queryClient.countUnique("<event_collection>", "<target_property>", new AbsoluteTimeframe("2015-05-15T19:00:00.000Z","2015-06-07T19:00:00.000Z"));
+double minimum = queryClient.minimum("<event_collection>", "<target_property>", new RelativeTimeframe("this_week"));
```

Comment:
Minor concern: technically, minimum/maximum/average/median can return null (if there are no events matching the timeframe & filters). This is a bit of an edge case, and maybe you're handling it some other way (e.g. by throwing an exception) somewhere else in the code, but it's something to think about.